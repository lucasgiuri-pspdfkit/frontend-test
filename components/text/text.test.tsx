import Text from "./text";
import { render } from "@testing-library/react";

const availableTypes = ["text", "title", "italic", "subtitle"];

describe("Text", () => {
  availableTypes.forEach((type) => {
    it(`Snapshot with type ${type}`, () => {
      const { container } = render(<Text type={type} text="test" />);
      expect(container).toMatchSnapshot();
    });
  });
});
