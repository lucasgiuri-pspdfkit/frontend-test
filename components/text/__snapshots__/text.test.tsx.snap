// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Snapshot with type italic 1`] = `
<div>
  <span
    class="italic font-semibold text-gray-400 text-sm"
  >
    test
  </span>
</div>
`;

exports[`Text Snapshot with type subtitle 1`] = `
<div>
  <h3
    class="text-lg font-bold text-gray-450"
  >
    test
  </h3>
</div>
`;

exports[`Text Snapshot with type text 1`] = `
<div>
  <span
    class=" font-semibold text-gray-400 text-sm"
  >
    test
  </span>
</div>
`;

exports[`Text Snapshot with type title 1`] = `
<div>
  <h2
    class="text-3xl font-bold text-gray-500"
  >
    test
  </h2>
</div>
`;
